mappings/net/minecraft/entity/projectile/Projectile.mapping

```diff
@@ -1,4 +1,13 @@
 CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/Projectile
+	FIELD field_22478 ownerUuid Ljava/util/UUID;
+	METHOD method_24919 setProperties (Lnet/minecraft/class_1297;FFFFF)V
+		ARG 1 user
+		ARG 2 pitch
+		ARG 3 yaw
+		ARG 4 roll
+		ARG 5 modifierZ
+		ARG 6 modifierXYZ
```

Comment:
Hmm, what are these modifiers? maybe multipliers?

Reply:
I just reused the names it had before 20w09a.